library/common/types/c_types.h

```diff
@@ -329,6 +337,7 @@ typedef struct {
  */
 typedef struct {
   envoy_log_f log;
```

Comment:
For parity, might want to consider calling this envoy_logger_log_f, but not a big deal.